### ISSUE TITLE
Add session support for service bus trigger

### ISSUE THREAD

**Issue by jeffhollan** [Azure/azure-functions-java-library#92]
Need a new property on the service bus trigger as session are supported in versions >= 3.1.0.

`isSessionsEnabled`

```json
    {
      "type": "serviceBusTrigger",
      "connection": "ServiceBusConnectionString",
      "isSessionsEnabled": true,
      "queueName": "queue",
      "name": "message"
    }
```

Also we don't need the `AccessRights` property. That's an appendage from v1 but isn't required in v2

**Comment by pragnagopa** [comment]
cc: @soninaren 
Since this will be a major version update for the bundles, we should address  issue https://github.com/Azure/azure-functions-extension-bundles/issues/2 and release a major version update of the azure-functions-java-library

**Comment by fabiocav** [comment]
@pragnagopa @jeffhollan @soninaren this has been sitting as a P1 for a while. Should it be assigned to the current sprint, or should we remove the label?

**Comment by jeffhollan** [comment]
It's not higher than the output binding stuff that Ahmed is doing now, but I do think it should be done very shortly after.  So I think P1 is fair (though maybe it has expired?)  At the very least do still think this is high priority.

**Comment by amamounelsayed** [comment]
Part of fix for #99 
https://github.com/microsoft/azure-maven-plugins/issues/822

**Comment by amamounelsayed** [comment]
function 1.3.5 has been released, updated function library to 1.3.1
